cfn/templates/ecs-task.template.yml

```diff
@@ -149,27 +149,21 @@ Resources:
   KlaxonTaskSchedule:
     Type: AWS::Events::Rule
     Properties:
-      # Description: String
-      # EventBusName: String # optional
-      # EventPattern: Json # optional
-      # Name: String
-      # RoleArn: String
-      # ScheduleExpression: String
-      # State: String
-      # Targets: 
-      #   - Target
       Description: Klaxon ecs task scheduler
       Name: klaxon-ecs-task-scheduler
-      RoleArn: !GetAtt ECSTaskExecutionRole.Arn
       ScheduleExpression: cron(0/10 * * * ? *)
       State: ENABLED
       Targets:
-        - Arn: !Ref KlaxonPageCheckTaskDefinition
+        - Arn: arn:aws:ecs:us-east-1:912288704264:cluster/newsroom-dev # cluster arn, hardcoding for now
           Id: klaxon-page-check-ecs-task
+          RoleArn: !GetAtt EC2ServiceEventsExecutionRole.Arn #ecs events role
+          EcsParameters:
+            TaskCount: 1
+            TaskDefinitionArn: !Ref KlaxonPageCheckTaskDefinition
```

Comment:
We're adjusting the target in order to closely align with event rule [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-target.html) example: 
```
MyEventsRule:
  Type: AWS::Events::Rule
  Properties:
    Description: Events Rule with EcsParameters
    EventPattern:
      source:
        - aws.ec2
      detail-type:
        - EC2 Instance State-change Notification
      detail:
        state:
          - stopping
    ScheduleExpression: rate(15 minutes)
    State: DISABLED
    Targets:
      - Arn: !GetAtt 
          - MyCluster
          - Arn
        RoleArn: !GetAtt 
          - ECSTaskRole
          - Arn
        Id: Id345
        EcsParameters:
          TaskCount: 1
          TaskDefinitionArn: !Ref MyECSTask
```

Reply:
Note how we must target a cluster as well as a ecs task